src/handlers/multiSig/helpers.ts

```diff
@@ -42,3 +53,68 @@ export const getUserMultiSigSignature = async ({
 
   return null;
 };
+
+export const getMultiSigFromDB = async (
+  databaseMultiSigId: string,
+): Promise<ColonyMultiSig | null | undefined> => {
+  const { data } =
+    (await query<GetColonyMultiSigQuery, GetColonyMultiSigQueryVariables>(
+      GetColonyMultiSigDocument,
+      {
+        id: databaseMultiSigId,
+      },
+    )) ?? {};
+
+  const multiSig = data?.getColonyMultiSig;
+
+  if (!multiSig) {
+    output(
+      'Could not find the multiSig in the db. This is a bug and needs investigating.',
+    );
+  }
+
+  return multiSig;
+};
+
+export const updateMultiSigInDB = async (
+  multiSigData: Partial<ColonyMultiSig> & { id: string },
+  showInActionsList?: boolean,
```

Comment:
I dislike that field for its lack of granularity, at the moment actions can be hidden for different reasons:
- motions below staking threshold (hardcoded in block-ingestor)
- if initiator/recipient is an extension
- if it's move funds to expenditure

But it's very hard to tell exactly why. It would be great for MultiSig not to use it.

If there's a use case to hide MultiSigs in UI, it would be preferred to store the data and let the client decide on filtering.

Reply:
fair point, I was replicating the way motions work, but honestly Multi Sig can just always show up. Will double check with product, but I think we can just automatically set this to true when it's created.